<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Figures</string>
    </property>
    <widget class="QRadioButton" name="BRectangle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BTriangle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Triangle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BCircle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Circle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BRhombus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Rhombus</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BHexagon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Hexagon</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BSquare">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Square</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BEllipse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Ellipse</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BStar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Star</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="BLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QLabel" name="squareLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="perimetrLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="xPointLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="yPointLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QRadioButton" name="BNew">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BMove1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BRotateL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BRotateR">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BMSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BPSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
